### ISSUE TITLE
Gradle wrapper

### ISSUE THREAD

**Issue by RoboMWM** [Rsl1122/Plan-PlayerAnalytics#889]
If you're going to remove maven (boooo), you should at least provide a gradle wrapper.

Though I hear providing a gradle wrapper is almost a necessity/recommended anyways since gradle updates too frequently.

**Comment by Rsl1122** [comment]
Gradle wrapper has been added by @Miclebrick

----

I switched out from maven because it broke my workflow due to dagger incompatibilities in multi-module projects.

I want to run tests before I commit to ensure that what I commit at least compiles.

**How maven was behaving**

- mvn test would fail due to small change in class that used dagger (for example removed constructor parameter)
- mvn clean had to be run to remove dagger generated classes
- mvn test fails because generated classes are not found for other modules since they haven't been generated
- mvn package has to be run
- mvn test finally runs, possibly failing

That was adding 5 extra minutes to testing every time I committed. Same issues were happening when I wanted to build an artifact. 

**How gradle works**

- I run gradle test and gradle takes care of the generated classes without issues.
